storage/storage-interface/src/cached_state_view.rs

```diff
@@ -199,17 +197,19 @@ impl TStateView for CachedStateView {
     fn get_state_value(&self, state_key: &StateKey) -> Result<Option<StateValue>> {
         let _timer = TIMER.with_label_values(&["get_state_value"]).start_timer();
         // First check if the cache has the state value.
-        if let Some(val_opt) = self.state_cache.get(state_key) {
+        if let Some(version_and_value_opt) =
```

Comment:
why not `if let Some((version, value_opt))`?

Reply:
Doesn't compile, it's a dashmap::Ref